src/improvements/single.just

```diff
@@ -20,11 +20,21 @@ simulate hdPath='0':
 
   config=${TASK_PATH}/config.toml
   script=${SCRIPT_PATH}/template/${SCRIPT_NAME}.sol
+  safe=$(bash ${SCRIPT_PATH}/script/get-safe.sh ${TASK_PATH} proxy-admin-owner)
 
   echo "RPC URL: ${rpcUrl}"
 
   echo "Using script ${script}"
   echo ""
+  signer=$(cast call ${safe} "getOwners()(address[])" -r ${rpcUrl} | grep -oE '0x[a-fA-F0-9]{40}' | head -n1)
```

Comment:
No following why we need this change. It's simulating correctly for me on main.

Reply:
Simulation works but it won't set the sender address as your own signer